configs/rover_configs.yaml

```
@@ -15,4 +15,21 @@
 # limitations under the License.
 #
 
-core:
\ No newline at end of file
+core:
+  client_grpc:
+    # The gRPC server address
+    server_addr: ${ROVER_CLIENT_GRPC_ADDR:localhost:11800}
```

```suggestion
  backend:
    # The gRPC server address
    addr: ${ROVER_CLIENT_GRPC_ADDR:localhost:11800}
```
I think this is more clear.